communication/message_formatting.md

```diff
@@ -4,12 +4,23 @@ Message Formatting
 - **3.5.1** ![](/badge/req.png) A client must encode emoticons as Unicode emoji.
 
 - **3.5.2** ![](/badge/req.png) A receiving client will make a best-effort to display
-  text exactly as it was sent, and must not transform the text in any way.
+  text exactly as it was intended, and must not materially change the text in any way.
```

Comment:
also, message splitting should allow the receiving client to recombine the message into one message, instead of 2 as it was sent.